modules/luci-mod-network/htdocs/luci-static/resources/view/network/wireless.js

```diff
@@ -944,6 +944,9 @@ return view.extend({
 					o = ss.taboption('general', form.Flag, 'legacy_rates', _('Allow legacy 802.11b rates'), _('Legacy or badly behaving devices may require legacy 802.11b rates to interoperate. Airtime efficiency may be significantly reduced where these are used. It is recommended to not allow 802.11b rates where possible.'));
 					o.depends({'_freq': '2g', '!contains': true});
 
+					o = ss.taboption('general', form.Flag, 'acs_exclude_dfs', _('Disable DFS'), _("Do not use DFS channels on auto selection"));
+					o.depends({'_freq': '5g', '_freq': 'auto', '!contains': true});
```

Comment:
This code makes no sense, the second `_freq` will overwrite the first one.

Reply:
@jow Thanks for your replay.
 I just noticed that when I selected 2.4 GHz and the frequency selection isset to `auto`. The `acs_exclude_dfs `option was still displayed. That is not what I want.

Is there a solution at all for this case?
I have not found anything in the entire LuCI repostiory.
Couldn't the validation be updated so that such a query works?
I am not quite sure what the data structure of _freq looks like.
It is obviously due to the special cbi view for _freq so I could not directly select the uci values.